### PR TITLE
Feat/alerts

### PR DESCRIPTION
**Pull request by TanmayJain92**
- Add Suppress Alerts for individual shipments.
- Add Refresh button on TopBar to fetch update existing and fetch new gateways. 
- Transmission and Measurement Time will be same for ProofTracker and ProoTracker Pro sensors.

### CODE REVIEW

File: src/pages/Shipment/CreateShipment.js [hunk 1]
```diff
@@ -548,6 +580,10 @@ const CreateShipment = ({ history, location }) => {
         max_excursion_humidity.setValue(value.max_excursion_humidity);
         shock_threshold.setValue(value.shock_threshold);
         light_threshold.setValue(value.light_threshold);
+        supressTempAlerts.setValue(value.alerts_to_suppress[0]);
+        supressHumidityAlerts.setValue(value.alerts_to_suppress[1]);
+        supressShockAlerts.setValue(value.alerts_to_suppress[2]);
+        supressLightAlerts.setValue(value.alerts_to_suppress[3]);
       }
```
Comment (patelradhika) [inline src/pages/Shipment/CreateShipment.js:587]: These check should be if includes and not by index

Comment (TanmayJain92) [reply src/pages/Shipment/CreateShipment.js:591]: Done.

File: src/pages/Shipment/CreateShipment.js [hunk 2]
```diff
@@ -548,6 +584,10 @@ const CreateShipment = ({ history, location }) => {
         max_excursion_humidity.setValue(value.max_excursion_humidity);
         shock_threshold.setValue(value.shock_threshold);
         light_threshold.setValue(value.light_threshold);
+        supressTempAlerts.setValue(!!_.includes(value.alerts_to_suppress, 'temperature'));
+        supressHumidityAlerts.setValue(!!_.includes(value.alerts_to_suppress, 'humidity'));
+        supressShockAlerts.setValue(!!_.includes(value.alerts_to_suppress, 'shock'));
+        supressLightAlerts.setValue(!!_.includes(value.alerts_to_suppress, 'light'));
       }
```
[the remark this answers is not in the document]
Comment (TanmayJain92) [reply src/pages/Shipment/CreateShipment.js:591]: Done

File: src/layout/TopBar/TopBar.js [hunk 3]
```diff
@@ -113,6 +121,12 @@ const TopBar = ({
     setAnchorEl(event.currentTarget);
   };
 
+  const handleFetchGatewaysClick = () => {
+    setRefreshClicked(true);
+    setAnchorEl(null);
+    queryClient.invalidateQueries({ queryKey: ['gateways', org_uuid] });
```
[the remark this answers is not in the document]
Comment (TanmayJain92) [reply src/layout/TopBar/TopBar.js]: Done

**Review by patelradhika — APPROVED** [review]
Looks good

**Pull request merged**